src/nodes/VectorNode.ts

```diff
@@ -56,6 +56,24 @@ export class VectorNode extends ArgumentNode implements ArrayLike<VectorElementN
         return ans
     }
 
+    getChunk() {
+        if (this.length !== 2 && this.length !== 3) {
+            return undefined
+        }
+        const vexX = this[0]
+        const vexZ = this.length === 3 ? this[2] : this[1]
+        // TODO: support VectorElementType.Relative
+        if (vexX.type != VectorElementType.Absolute || vexZ.type != VectorElementType.Absolute) {
```

Comment:
```suggestion
        if (vexX.type !== VectorElementType.Absolute || vexZ.type !== VectorElementType.Absolute) {
```

I personally prefer using `!==` everywhere it can be used to keep it consistent.